### PR TITLE
Replaced Godeps with glide

### PR DESCRIPTION
**Pull request by tboerger**
The same as for the other appc repositories. I have entirely replaced Godeps with glide and while I have imported Godeps into glide it still updated some dependencies.

### CODE REVIEW

**Comment by jonboulle** [comment]
Same as elsewhere - this imports a bunch of cruft - could you run glide-vc please?

**Comment by tboerger** [comment]
> Same as elsewhere - this imports a bunch of cruft - could you run glide-vc please?

Did that now, thousands of changes lesser now.

**Comment by jonboulle** [comment]
how about with the --no-tests flag? :-)

**Comment by tboerger** [comment]
`glide-vc --no-tests --only-code` is a working combination. Applied it.

**Comment by s-urbaniak** [comment]
@tboerger thanks! Just a few nitpicks from my side:
- travis is failing, because the `build` script was renamed to `build.sh` in your PR.
- Do you mind to add a `glide-update.sh` script, so others can bump dependencies in a reproducible way (see https://github.com/coreos/rkt/blob/master/scripts/glide-update.sh)?
- I am wondering if we should document the glide update process, maybe in `CONTRIBUTING.md`?

**Comment by tboerger** [comment]
Except the documentation on `CONTRIBUTING.md` I have applied the comments.

**Comment by jonboulle** [comment]
@s-urbaniak rereview?

File: glide.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package: "github.com/appc/spec"
+import:
+- package: github.com/coreos/go-semver
+  version: 6fe83ccda8fb9b7549c9ab4ba47f47858bc950aa
```
Comment (s-urbaniak) [inline glide.yaml]: "v0.1.0"

File: glide.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  subpackages:
+  - semver
+- package: github.com/gogo/protobuf
+  version: 4168943e65a2802828518e95310aeeed6d84c4e5
```
Comment (s-urbaniak) [inline glide.yaml]: "v0.2"

File: glide.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  subpackages:
+  - errorutil
+- package: golang.org/x/net
+  version: ccfcd82c7124abd517842acbacc3b8c1e390c73d
+  subpackages:
+  - html
+  - html/atom
+- package: gopkg.in/inf.v0
+  version: 3887ee99ecf07df5b447e9b00d9c0b2adaa9f3e4
+- package: k8s.io/kubernetes
```
Comment (s-urbaniak) [inline glide.yaml]: "v1.3.0"

**Comment by s-urbaniak** [comment]
@tboerger added a few version pointers for the glide.yaml, where available. We bumped to k8s 1.3.0 in master, so you'll have to revendor and rebase.

**Comment by tboerger** [comment]
@s-urbaniak I think now it should be fine :)

**Comment by s-urbaniak** [comment]
great, thanks, hopefully you won't kill me now if I ask you to reformat the commit messages to comply to https://github.com/appc/spec/blob/master/CONTRIBUTING.md#format-of-the-commit-message ;-)

**Comment by tboerger** [comment]
@s-urbaniak hopefully you are happy now? :)

**Comment by s-urbaniak** [comment]
@tboerger good stuff, I checked out the PR locally, and found one thing, in commit https://github.com/appc/spec/pull/632/commits/29e4e7af3523fe479f55127a3d180b11672227ff you're pulling all vendored ~140MB dependencies, and in commit https://github.com/appc/spec/pull/632/commits/f824d7b9a73a2c8ebb9809311da08305614acef4 you are pruning them down to ~1MB.

I'd suggest to squash both commits (check in only pruned dependencies using `glide-vc`), that way we don't leave ~140MB of unnecessary blobs in the git history.

Other than that lgtm, and thanks a lot for the patience ;-)

**Comment by tboerger** [comment]
@s-urbaniak done :)

**Comment by s-urbaniak** [comment]
@tboerger thanks a lot, lgtm! :+1:

**Comment by jonboulle** [comment]
Outstanding. Thanks so much!

**Pull request merged**